docs/operations/updating_the_stack.rst

```diff
@@ -86,7 +86,21 @@ You can copy down your ``cops.pem`` into your ``~/.ssh`` from bastion2 by:
 Steps
 *****
 
-1. Set Up SSH Tunnel
+1. Update your buildout and JS dependencies
+===========================================
+
+Keep sure you are on the latest master branch.
+
+Update/install JS libraries regularly for the fly command later:
+
+.. code-block:: bash
+
+   cd bakery
+   # run yarn or npm install
+   yarn
```

Comment:
let's default to `npm install`, since this repo uses `package-lock.json`, although `yarn` (v1) is acceptable. I believe this repo is not compatible with yarn berry (v2) in case someone is using that.
```suggestion
   npm install
```